client/lib/abtest/active-tests.js

```diff
@@ -3,10 +3,10 @@ export default {
 	improvedOnboarding: {
 		datestamp: '20190214',
 		variations: {
-			main: 50,
-			onboarding: 50,
+			main: 0,
+			onboarding: 100,
 		},
-		defaultVariation: 'main',
```

Comment:
We should leave this as `main` - changing it means that we'd be [assigning `onboarding` to non-en and existing users](https://github.com/Automattic/wp-calypso/blob/master/client/lib/abtest/index.js#L148).

Reply:
Done!